package.json

```diff
@@ -22,63 +22,60 @@
     "url": "https://github.com/SebastianM/angular2-google-maps/issues"
   },
   "homepage": "https://github.com/SebastianM/angular2-google-maps#readme",
-  "dependencies": {
-    "@angular/common": "2.0.0-rc.2",
-    "@angular/compiler": "2.0.0-rc.2",
-    "@angular/core": "2.0.0-rc.2",
-    "@angular/platform-browser": "2.0.0-rc.2",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.2",
-    "es6-shim": "^0.35.0",
-    "reflect-metadata": "^0.1.3",
-    "rxjs": "5.0.0-beta.6",
-    "zone.js": "^0.6.12"
-  },
-  "devDependencies": {
-    "babel-eslint": "6.0.4",
-    "clang-format": "1.0.41",
-    "del": "2.2.0",
-    "eslint": "^2.10.2",
+  "dependencies": {},
+  "peerDependencies": {
```

Comment:
this is done by the build process. for development, it makes sense to have them as dependencies.